### PR TITLE
SSJ - Scenario Outline: Verifying applicant is able to see "Your Applications" tab and page content after applying to a Vacancy

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

**Review by iamdez99 — COMMENTED** [review]
Reviewed

File: src/main/java/com/nci/automation/web/CommonUtils.java [hunk 1]
```diff
@@ -790,11 +854,11 @@ public static String getDateOneMonthFromNowIn_YYYY_MM_DD_format() {
     /**
      * Compares two lists of elements and asserts that they are equal.
      *
-     * @param actualValues The list of actual values.
+     * @param actualValues   The list of actual values.
```
Comment (sarwarahmed1) [inline src/main/java/com/nci/automation/web/CommonUtils.java:857]: Is there a reason for this extra space here?

**Review by sarwarahmed1 — APPROVED** [review]
Approved!

**Review by lfcabrera14 — APPROVED** [review]
Reviewed and approved

**Review by andreyeve — APPROVED** [review]

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved! Great work!

**Review by puzikovam2 — APPROVED** [review]
approved by @puzikovam2

**Pull request merged**